mobile/lib/shared/providers/server_info.provider.dart

```diff
@@ -15,13 +15,36 @@ class ServerInfoNotifier extends StateNotifier<ServerInfoState> {
               patch_: 0,
               minor: 0,
             ),
+            serverFeatures: ServerFeaturesDto(
+              clipEncode: true,
+              configFile: false,
+              facialRecognition: true,
+              map: true,
+              oauth: false,
+              oauthAutoLaunch: false,
+              passwordLogin: true,
+              search: true,
+              sidecar: true,
+              tagImage: true,
+            ),
+            serverConfig: ServerConfigDto(
+              loginPageMessage: "",
+              mapTileUrl: "https://tile.openstreetmap.org/{z}/{x}/{y}.png",

```

Comment:
the hardcoded URL before was `"https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"`. The new one is missing `{s}` is this correct?

Reply:
Yes. The template `{s}` was previously used to handle subdomains for openstreetmap. But since we are accepting tile urls as user inputs, we can no longer determine the subdomains on which they are served. Users can now configure the url more directly as `https://a.tile.openstreemap.org/{z}/{x}/{y}.png` or other similar subdomains instead